### PR TITLE
Fix footer text and icons visibility on light mode

### PR DESCRIPTION
**Pull request by mativanarquero**
- [x] Fix visibility - Update footer custom links to static white on dark/light mode 
- [x] Fix visibility - Inline  styling for custom icons links 
- [x] Update header text - Rename 'crypto currency' to 'cryptocurrency'
- [x] Re-Update project layout container height from 14vh to 10vh
project layout container height

footer text visibility and header body

### CODE REVIEW

File: src/components/layout/Page.tsx [hunk 1]
```diff
@@ -59,9 +62,9 @@ const PageFooter = () => {
             <Flex flexDirection="column" justifyContent="space-evenly" style={{ textAlign: 'left' }}>
                 <img src={Powered} alt="powered-logo" style={{ width: '60%', margin: '20px' }} />
                 <Flex justifyContent="space-evenly" style={{ width: '100%' }}>
-                    <CustomLink href="#">Terms & Conditions</CustomLink>
+                <CustomLink href="#"><h5>Terms & Conditions</h5></CustomLink>
```
Comment (Pseudoman21) [inline src/components/layout/Page.tsx:65]: Nahale mo ata su tab Mat hehe

Comment (mativanarquero) [reply src/components/layout/Page.tsx:65]: Ay witz haha

Comment (mativanarquero) [reply src/components/layout/Page.tsx]: Oki na jeyps pa check po

Comment (Pseudoman21) [reply src/components/layout/Page.tsx]: Nice! Iyu okay na yan hehe

**Review by harveyjavier — APPROVED** [review]
Lights on, lights off!

**Review by Pseudoman21 — APPROVED** [review]
Adaptive!

**Review by swengr-janan — CHANGES_REQUESTED** [review]
image

@mativanarquero , may reason ba kung bakit di mo nilagyan h5 tag si Privacy Policy? Kung wala naman, pa lagay nalang din for unifomnity

**Comment by harveyjavier** [comment]
> image
> 
> @mativanarquero , may reason ba kung bakit di mo nilagyan
> 
> ##### tag si Privacy Policy? Kung wala naman, pa lagay nalang din for unifomnity

Pati terms. Though pwede rin natin yan gawan

**Comment by harveyjavier** [comment]
Nice balpz! 🦅👀 talaga! haha

**Comment by swengr-janan** [comment]
> > image
> > @mativanarquero , may reason ba kung bakit di mo nilagyan
> > ##### tag si Privacy Policy? Kung wala naman, pa lagay nalang din for unifomnity
> 
> Pati terms. Though pwede rin natin yan gawan

Nice! Update nalang sa link @harveyjavier . Thanksu!!

**Comment by harveyjavier** [comment]
Yup, update nlng natin si link

Si link:
image

**Review by swengr-janan — APPROVED** [review]
Update nalang kita @mativanarquero kung may link na sa Terms and Privacy. Approve ko na to pero di ko muna i close si issue.

**Comment by mativanarquero** [comment]
@swengr-janan @harveyjavier Nasisira po ang styling if inline ang styling ng Privacy Policy kaya sinet ko nalang sa StyledLink (which is yung nag wra-wrap sakanya) ang color. 
styledlink
privacy styling

**Comment by mativanarquero** [comment]
> Update nalang kita @mativanarquero kung may link na sa Terms and Privacy. Approve ko na to pero di ko muna i close si issue.

Copy sir. Thank you.

**Comment by mativanarquero** [comment]
> image
> 
> @mativanarquero , may reason ba kung bakit di mo nilagyan h5 tag si Privacy Policy? Kung wala naman, pa lagay nalang din for unifomnity

Ay witz yung link ba? HAHAHA mali ata basa ku

**Pull request merged**